apps/payments/next/.env.production

```diff
@@ -24,9 +24,13 @@ MYSQL_CONFIG__ACQUIRE_TIMEOUT_MILLIS=
 # GeoDBConfig
 GEODB_CONFIG__DB_PATH=
 
+# StripeConfig
 STRIPE_CONFIG__API_KEY=11233
 STRIPE_CONFIG__TAX_IDS={}
 
+# Stripe
+STRIPE_PUBLIC_API_KEY=
+
```

Comment:
Nit: similar to how you added `# StripeConfig` on L27, could you add `# PayPal Config`?

Reply:
Nit: I think this was missed.

Reply:
I think this was missed.